pkg/imported/components/VersionManagement.vue

```diff
@@ -0,0 +1,118 @@
+<script>
+import { defineComponent } from 'vue';
+import { RadioGroup } from '@components/Form/Radio';
+import { mapGetters } from 'vuex';
+import { _EDIT } from '@shell/config/query-params';
+import Banner from '@components/Banner/Banner.vue';
+const VERSION_MANAGEMENT_DEFAULT = 'system-default';
+
+export default defineComponent({
+  emits: ['version-management-changed'],
+
+  components: { RadioGroup, Banner },
+
+  props: {
+    value: {
+      type:     String,
+      required: true
+    },
+    mode: {
+      type:     String,
+      required: true,
+    },
+    globalSetting: {
+      type:     Boolean,
+      required: true,
+    },
+    oldValue: {
+      type:     String,
+      required: true
+    },
+
+  },
+
+  computed: {
+    ...mapGetters({ t: 'i18n/t', features: 'features/get' }),
+    isEdit() {
+      return this.mode === _EDIT;
+    },
+    versionManagementOptions() {
+      const defaultLabel = this.globalSetting ? this.t('imported.basics.versionManagement.globalEnabled') : this.t('imported.basics.versionManagement.globalDisabled');
+
+      return [
+        { value: VERSION_MANAGEMENT_DEFAULT, label: defaultLabel },
+        { value: 'true', label: this.t('imported.basics.versionManagement.enabled') },
+        { value: 'false', label: this.t('imported.basics.versionManagement.disabled') },
+      ];
+    },
+    versionManagementInfo() {
+      if (!this.isEdit) {
+        if ( this.value === VERSION_MANAGEMENT_DEFAULT ) {
+          return this.t('imported.basics.versionManagement.banner.create.default');
+        } else {
+          return this.t('imported.basics.versionManagement.banner.create.nonDefault');
+        }
+      } else {
+        if (this.oldValue === VERSION_MANAGEMENT_DEFAULT) {
+          return this.value === this.globalSetting ? this.t('imported.basics.versionManagement.banner.edit.defaultToNonDefault', {}, true) : `${ this.t('imported.basics.versionManagement.banner.edit.different') } ${ this.t('imported.basics.versionManagement.banner.edit.defaultToNonDefault', {}, true ) }`;
+        } else {
+          if (this.value === VERSION_MANAGEMENT_DEFAULT) {
+            return this.oldValue === this.globalSetting ? this.t('imported.basics.versionManagement.banner.edit.nonDefaultToDefault', {}, true) : `${ this.t('imported.basics.versionManagement.banner.edit.different') } ${ this.t('imported.basics.versionManagement.banner.edit.nonDefaultToDefault', {}, true ) }`;
+          } else {
+            return this.t('imported.basics.versionManagement.banner.edit.different');
+          }
+        }
+      }
+    },
+
+    globalConfigurationText() {
+      return this.globalSetting ? this.t('imported.basics.versionManagement.summary.globallyEnabled', {}, true) : this.t('imported.basics.versionManagement.summary.globalDisabled', {}, true);
```

Comment:
the disabled text is not rendering, looks like the right key is `globallyDisabled`